config/config.ini.example

```diff
@@ -25,7 +25,10 @@
 #show-all-zoom-level            # Show all Pokemon, even excluded, at this map zoom level.
                                 # Set to 0 to disable this feature. Set to 19 or higher for nice results.
 #map-title:                     # Title of the map. (default='RocketMap')
-#paypal-url:                    # PayPal donation link. (default=None)
+#header-image:                  # Image in header. Place image in 'static/images/header'. (default='rocket.png')
```

Comment:
Nice feature :+1: 
With the possibility to change the favicon it would be complete (I think)